app/ui-react/syndesis/src/modules/connections/pages/create/ConfigurationPage.tsx

```diff
@@ -124,24 +152,9 @@ export const ConfigurationPage: React.FunctionComponent = () => {
                 >
                   {() =>
                     acquisitionMethod ? (
-                      <ConnectorAuthorization
-                        i18nTitle={t(
-                          'connections:create:configure:configurationTitle',
-                          { name: connector.name }
-                        )}
-                        i18nDescription={t(
-                          'connections:create:configure:configurationDescription',
-                          { name: connector.name }
-                        )}
-                        i18nConnectButton={t(
-                          'connections:create:configure:configurationButton',
-                          { name: connector.name }
-                        )}
-                        isConnecting={isConnecting}
-                        onConnect={connectOAuth}
-                      />
+                      <OAuthFlow connector={connector} history={history} />
                     ) : (
-                      <WithConnectorForm connector={connector} onSave={onSave}>
+                      <WithConnectorForm connector={connector} onSave={onSaveTFunction}>
```

Comment:
fat fingered?

Reply:
Totally, and I did fix that, I wonder how I commited the unfixed version though... thanks for finding this :)